shell/assets/translations/ru-ru.yaml

```
@@ -6642,472 +6636,470 @@ typeLabel:
+  unassign: "Отменить назначение"
+  uninstall: Удалить
+  redeploy: Повторное развёртывание
+  addSidecar: Добавить Sidecar
+  rollback: Откатить
+  openShell: Выполнить Shell
+  runNow: Запустить сейчас
+  suspend: Приостановить
+  resume: Продолжить
 
 unit:
-  sec: secs
-  min: mins
+  sec: сек.
+  min: мин.
   hour: |-
     {count, plural,
-      one { hour }
-      other { hours }
+      one { час }
+      other { ч. }
     }
   day: |-
     {count, plural,
-      one { day }
-      other { days }
+      one { день }
+      other { д. }
     }
 workloadPorts:
-  addPort: Add Port or Service
-  remove: Remove
-  addHost: Add Host
+  addPort: Добавить порт или сервис
+  remove: Удалить
+  addHost: Добавить хост
 
 podAffinity:
-  addLabel: Add Pod Selector
+  addLabel: Добавить селектор подов
 
 keyValue:
-  keyPlaceholder: e.g. foo
-  valuePlaceholder: e.g. bar
-  protip: 'Paste lines of <em>key=value</em> or <em>key: value</em> into any key field for easy bulk entry'
+  keyPlaceholder: например, foo
+  valuePlaceholder: например, bar
+  protip: "Вставьте строки вида <em>ключ=значение</em> или <em>ключ: значение</em> в любое ключевое поле для упрощения массового ввода"
 
 registryMirror:
-  header: Mirrors
-  toolTip: 'Mirrors can be used to redirect requests for images from one registry to come from a list of endpoints you specify instead.  For example docker.io could redirect to your internal registry instead of ever going to DockerHub.'
-  addLabel: Add Mirror
-  description: Mirrors define the names and endpoints for private registries. The endpoints are tried one by one, and the first working one is used.
-  hostnameLabel: Registry Hostname
-  hostnamePlaceholder: e.g. docker.io or *
-  endpointsLabel: Mirror Endpoints
-  endpointsPlaceholder: e.g. a.registry.com:5000, b.registry.com:5000
+  header: Зеркала
+  toolTip: "Зеркала можно использовать для перенаправления запросов, чтобы, вместо данного регистри, ответы на запросы образов приходили от endpoint'ов из заданного списка. Например, docker.io может всегда перенаправлять на внутренний регистри вместо обращения к DockerHub."
+  addLabel: Добавить зеркало
+  description: Зеркала определяют имена и endpoint'ы для приватных регистри. Эти endpoint'ы проверяются по-очереди, и используется первый работающий.
+  hostnameLabel: Имя хоста регистри
+  hostnamePlaceholder: например, docker.io или *
+  endpointsLabel: Endpoint'ы зеркала
+  endpointsPlaceholder: например, a.registry.com:5000, b.registry.com:5000
 
 registryMirrorRewrite:
   header: Rewrites
-  toolTip: 'Each mirror can have a set of rewrites. Rewrites can change the tag of an image based on a regular expression.'
-  addLabel: Add Rewrite Config
-  keyLabel: Rewrite pattern
-  keyPlaceholder: e.g. ^rancher/(.*)
-  valueLabel: Rewrite replacement
-  valuePlaceholder: e.g. mirrorproject/rancher-images/$1
+  toolTip: "Каждое зеркало может иметь набор rewrites. Rewrites могут изменять тэг образа на основе регулярного выражения."
```

```suggestion
  toolTip: "Каждое зеркало может иметь набор rewrites. Rewrites могут изменять тег образа на основе регулярного выражения."
```
Можно и "тэг", но чаще по тексту мы использовали "тег" пусть будет один вариант